app/src/main/java/io/mercury/coroutinesandbox/interactors/GetDownloadedMovies.kt

```diff
@@ -0,0 +1,22 @@
+package io.mercury.coroutinesandbox.interactors
+
+import io.mercury.coroutinesandbox.repos.MovieDownloadManager
+import io.mercury.domain.interactors.GetMovies
+import io.mercury.domain.models.Movie
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class GetDownloadedMovies @Inject constructor(
+    private val downloadManager: MovieDownloadManager,
+    private val getMovies: GetMovies
+) {
+    operator fun invoke(): Flow<List<Movie>> {
+        val allMovies = getMovies.invoke()
```

Comment:
I may have asked this before, but I'm not able to find the comment. 

Should we be using `getMovies()` vs. `getMovies.invoke()`? Or is there are reason to be calling `invoke()` directly?

Reply:
This was an oversight on my part. No reason that I am aware of, so I'll fix.